names/names.py

```diff
@@ -11,12 +11,19 @@
 f.close()
 
 duplicates = []
-for name_1 in names_1:
-    for name_2 in names_2:
-        if name_1 == name_2:
-            duplicates.append(name_1)
+# for name_1 in names_1:
+#     for name_2 in names_2:
+#         if name_1 == name_2:
+#             duplicates.append(name_1)
+
+
+
+duplicates = list(set(names_1).intersection(names_2))
```

Comment:
Hmmm what's the run time on this? Double check the run time of `intersection` and compare that. You had the first step right by converting it into a set, that's what I would've done so you can look up O(1) time.

```
duplicates = []
names_2_set = set(names_2) #O(1)
for name in names_1: #O(n)
  if name in names_2_set: #O(1)
    duplicates.append(name) #O(1)
```
So this run time would be O(n)

I believe yours would still be O(n^2). Let me know if I am wrong